docs/themes/book/layouts/shortcodes/project-setup.html

```diff
@@ -0,0 +1,21 @@
+In the `settings.gradle`:
+
+```groovy
+pluginManagement {
+    repositories {
+        maven {
+            name = "Avito bintray"
+            url = "https://dl.bintray.com/avito/maven"
+        }
+    }
+    resolutionStrategy {
+        eachPlugin {
+            String pluginId = requested.id.id
+            if (pluginId.startsWith("com.avito.android")) {
+                def artifact = pluginId.replace("com.avito.android.", "")
+                useModule("com.avito.android:$artifact:$avitoToolsVersion")
+            }
+        }
+    }
+}
+```
```

Comment:
Let's move this file to `docs/layouts/shortcodes` if it's not a part of the book theme.

Reply:
What is it? Is the shortcode some concept of our theme?

Reply:
Shortcode is a Hugo feature. Theme mostly mirrors a structure. Thus it brings its own resources and can override them.

https://gohugo.io/templates/shortcode-templates/#shortcode-template-lookup-order

Reply:
moved